data/json/requirements/toolsets.json

```
@@ -216,6 +216,42 @@
       { "id": "SCREW", "level": 1 }
     ]
   },
+  {
+    "id": "advanced_electronics_low_voltage",
+    "type": "requirement",
+    "//": "For electronics and electrical terrain with voltage low enough it's going to mildly inconvenience you, includes wirecutting and pliers",
+    "tools": [
+      [
+        [ "hacksaw", -1 ],
+        [ "multitool", -1 ],
+        [ "boltcutters", -1 ],
+        [ "toolset", -1 ],
+        [ "toolbox", -1 ],
+        [ "toolbox_workshop", -1 ]
+      ],
+      [ [ "multitool", -1 ], [ "pliers", -1 ], [ "toolset", -1 ] ]
+    ],
+    "qualities": [ { "id": "PRY", "level": 2 }, { "id": "SCREW_FINE", "level": 1 }, { "id": "WRENCH_FINE", "level": 1 } ]
+  },
+  {
+    "id": "advanced_electronics_high_voltage",
+    "type": "requirement",
+    "//": "For electronics and electrical terrain with voltage so high it kills you just for looking at it funny, includes wirecutting and pliers",
+    "tools": [
+      [
+        [ "hacksaw", -1 ],
+        [ "multitool", -1 ],
+        [ "boltcutters", -1 ],
+        [ "toolset", -1 ],
+        [ "toolbox", -1 ],
+        [ "survivor_belt", -1 ],
+        [ "toolbox_workshop", -1 ]
+      ],
+      [ [ "multitool", -1 ], [ "pliers", -1 ], [ "toolset", -1 ] ],
```

Per discussion in the BN server
```suggestion
      [ [ "multitool", -1 ], [ "pliers", -1 ], [ "toolset", -1 ] ],
      [ [ "voltmeter", 10 ] ],
```

---

Looks good